tests/symbol_test.py

```diff
@@ -0,0 +1,34 @@
+from .context import assert_equal
+import pytest
+from sympy import Symbol, simplify
+from latex2sympy.latex2sympy import process_sympy
+
+
+def test_symbol_parses_as_positive():
+    assert_equal('x', Symbol('x', real=True, positive=True))
+
+
+def test_symbol_equivalence():
```

Comment:
@builtbywill Would it be useful to have a test where if `positive=True` is not specified, that the equivalence fails? Perhaps useful to catch future regressions? 🤷

Reply:
At least as far as this test goes, if positive is not True, then not all of them pass, so maybe that is enough?

Reply:
Indeed! :thumps: